docs/core/docker/visual-studio-tools-for-docker.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Visual Studio Tools for Docker
+description: Using Visual Studio Tools for Docker 
+keywords: .NET, .NET Core, Docker, ASP.NET Core, Visual Studio 2015
+author: spboyer
+manager: wpickett
+ms.date: 09/16/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 1f3b9a68-4dea-4b60-8cb3-f46164eedbbf
+---
+
+# Visual Studio Tools for Docker 
+Developing and debugging your application in a Docker container can be a ceremony of tasks to get setup with various tools. [Visual Studio Tools for Docker](https://visualstudiogallery.msdn.microsoft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4) helps you get past the hurdles and find the bugs using F5 to debug your application directly in a locally hosted Docker Container. 
+
+> [!NOTE]
+>The current version targets Linux Docker containers, with Windows Containers coming soon.
+
+## Prerequisites
+- [Microsoft Visual Studio 2015 Update 3](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- [Microsoft .NET Core 1.0.1 RTM VS 2015 Tooling Preview 2, with the 1.0.1 updates](https://go.microsoft.com/fwlink/?LinkID=827546)
```

Comment:
I was mostly thinking about the right branding which would be ".NET Core 1.0.1 - VS 2015 Tooling Preview 2".